arm-monitor/2017-03-01/swagger/activityLogAlerts_API.json

```diff
@@ -0,0 +1,451 @@
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/ActivityLogAlertList"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the list of activity log alerts could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Resource": {
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "Azure resource Id"
+                },
+                "name": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "Azure resource name"
+                },
+                "type": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "Azure resource type"
+                },
+                "location": {
+                    "type": "string",
+                    "description": "Resource location",
+                    "x-ms-mutability": [
+                        "create",
+                        "read"
+                    ]
+                },
+                "tags": {
+                    "additionalProperties": {
+                        "type": "string"
+                    },
+                    "description": "Resource tags"
+                }
+            },
+            "required": [
+                "location"
+            ],
+            "x-ms-azure-resource": true,
+            "description": "An azure resource object"
+        },
+        "ActivityLogAlertResource": {
+            "description": "An activity log alert resource.",
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ],
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ActivityLogAlert"
+                }
+            }
+        },
+        "ActivityLogAlertList": {
+            "description": "A list of activity log alerts.",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ActivityLogAlertResource"
```

Comment:
This is correct as written the items in the list are of type ActivityLogAlertResource